pkg/controller/gcp/compute/gke.go

```diff
@@ -251,12 +260,9 @@ func (r *Reconciler) Reconcile(request reconcile.Request) (reconcile.Result, err
 	instance := &gcpcomputev1alpha1.GKECluster{}
 	err := r.Get(ctx, request.NamespacedName, instance)
 	if err != nil {
-		if errors.IsNotFound(err) {
-			// Object not found, return.  Created objects are automatically garbage collected.
-			// For additional cleanup logic use finalizers.
+		if kerrors.IsNotFound(err) {
 			return reconcile.Result{}, nil
 		}
-		// Error reading the object - requeue the request.
 		return reconcile.Result{}, err
```

Comment:
Do we really not want to requeue here, or should this be a RequeueOnFailure scenario?

Reply:
This is similar to the case above: the explicit `requeue` in this case is not needed since the object will be requeued due to the error.